src/jit/emitxarch.cpp

```diff
@@ -4326,6 +4330,67 @@ void emitter::emitIns_R_R_AR(instruction ins, emitAttr attr, regNumber reg1, reg
     emitCurIGsize += sz;
 }
 
+// return true if ins is an AVX2 Gather instruction
```

Comment:
Since this is new, you should add a header here as well.

Reply:
Will do, thanks.